xbmc/filesystem/AddonsDirectory.cpp

```diff
@@ -787,10 +787,12 @@ void CAddonsDirectory::GenerateAddonListing(const CURL &path,
     bool installed = CServiceBroker::GetAddonMgr().IsAddonInstalled(addon->ID());
     bool disabled = CServiceBroker::GetAddonMgr().IsAddonDisabled(addon->ID());
     bool hasUpdate = outdated.find(addon->ID()) != outdated.end();
+    bool fromOfficialRepo = CServiceBroker::GetAddonMgr().IsFromOfficialRepo(addon);
```

Comment:
@phunkyfish how do we adjust this call when the function is static and removed from `AddonManager`

Reply:
Exactly right!